src/plugins/biostruct3d_view/src/deprecated/BioStruct3DViewPlugin.cpp

```diff
@@ -93,8 +93,12 @@ extern "C" Q_DECL_EXPORT QString *U2_PLUGIN_FAIL_MASSAGE_FUNC() {
 BioStruct3DViewPlugin::BioStruct3DViewPlugin()
     : Plugin(tr("3D Structure Viewer"), tr("Visualizes 3D structures of biological molecules.")) {
     // Init plugin view context
-    viewContext = new BioStruct3DViewContext(this);
-    viewContext->init();
+    if (BioStruct3DGLWidget::canRender()) {
```

Comment:
For reviewers: should this check be enabled only on Windows systems and not on Mac or Linux?

Reply:
It is worth testing this code on Mac and Linux

Reply:
Could you please run Linux Nighly for your branch to see that non of the existing 3D-related tests are broken? (you will need to do it anyway to submit any CL)